### PR TITLE
Update microsoft-365-business-faqs.md

### PR DESCRIPTION
**Pull request by msbemba**
added conditional access feature

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/business/support/microsoft-365-business-faqs.md [hunk 1]
```diff
@@ -40,6 +40,9 @@ Microsoft 365 is an integrated solution that brings together best-in-class produ
 * Always up-to-date Office + Windows 10 
 * Streamlined deployment of PCs with Windows AutoPilot
 
+**Enabling Conditional Access**
+- Microsoft 365 Business customers can enable Conditional Access via the Azure Directory settings in the Azure portal. For more information on how to configure Conditional Access policies, please see the article [What is Conditional access](https://docs.microsoft.com/en-us/azure/active-directory/conditional-access/overview).
```
Comment (JohanFreelancer9) [inline microsoft-365/business/support/microsoft-365-business-faqs.md]:
```suggestion
- Microsoft 365 Business customers can enable Conditional Access via the Azure Directory settings in the Azure portal. For more information on how to configure Conditional Access policies, please see the article [What is Conditional access](https://docs.microsoft.com/azure/active-directory/conditional-access/overview).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review - please remove locale from link.

**Comment by mypil** [comment]
@Sirkkuw - PR has been copy edited and is ready for final review. This is based on issue #461.

Thank you.

**Comment by Sirkkuw** [comment]
@mypil go ahead and merge.

**Comment by mypil** [comment]
@Sirkkuw - Please assist to merge this. Thank you.

**Pull request merged**